app/Helpers/Util.php

```diff
@@ -175,4 +178,17 @@ public static function generateShortCode($strength = 9) {
         return  "can-" . $random_string;
     }
 
+    public static function topicHistoryLink($topicNum, $campNum = 1, $title, $campName = 'Aggreement' , $type)
+    {
+        $title = preg_replace('/[^A-Za-z0-9\-]/', '-', $title);
+        $campName = preg_replace('/[^A-Za-z0-9\-]/', '-', $campName);
+        $topicId = $topicNum . "-" . $title;
+        $campId = $campNum . "-" . $campName;
+        if($type == "topic"){
```

Comment:
`if() else {}` can be replaced as

` $word = ($type == "topic") ? 'topic' : 'camp';`

And then use single return statement.